src/apps/patron-page/util/BranchesDropdown.tsx

```diff
@@ -28,7 +28,7 @@ const BranchesDropdown: FC<BranchesDropdownProps> = ({
         htmlFor="branches-dropdown"
         className="text-body-medium-medium mt-32 mb-8"
       >
-        {`${t("pickupBranchesDropdownLabelText")}${required ? "*" : null}`}
+        {`${t("pickupBranchesDropdownLabelText")}${required ? " *" : ""}`}
```

Comment:
I think it could be nice to make an atom component (maybe organised in the atom compinent folder in a sub flder called forms?).
It could be a component that receives two props:
required: boolean
text: string
That would make this place and the `TextInput`label place look more neat I think.